jupyterhub/templates/hub/deployment.yaml

```diff
@@ -45,11 +45,6 @@ spec:
                     - key: release
                       operator: In
                       values: [{{ .Release.Name | quote }}]
-      {{- $_ := merge (dict "podKind" "core") . }}
-      {{- $dummy := include "jupyterhub.prepareScope" $_ }}
-      {{- if $_.hasTolerations }}
-      {{- include "jupyterhub.tolerations" $_ | nindent 6 }}
-      {{- end }}
```

Comment:
There was a hatch for tolerations on core nodes, but it was not adding anything by default anyhow.